### PR TITLE
replaced `cref="..."` with cref="@...`

### PR DESCRIPTION
**Pull request by eaba**
Part fix for #5542

@Aaronontheweb 

This [log](https://dev.azure.com/dotnet/fa49bc58-13a9-42fb-8dd1-307759222d95/_apis/build/builds/65510/logs/137) shows warnings about `Invalid cref value "!:xxxxxxx" found in triple-slash-comments....` which I have not been able to reproduce locally. Locally the warnings are down to 201 with majority of the WARNINGS being `Could not resolve base documentation for`.

I found this comment interesting https://github.com/dotnet/docfx/issues/5112#issuecomment-568178162 @Arkatufus

### CODE REVIEW

File: output.txt [hunk 1]
```diff
@@ -0,0 +1,2497 @@
+
+C:\Users\Ebere\source\repos\akka.net>PowerShell.exe -file "build.ps1" docfx --warningsAsErrors
```
[the remark this answers is not in the document]
Comment (Aaronontheweb) [reply output.txt]: i.e. the entire file needs to be deleted

**Comment by Aaronontheweb** [comment]
I just found out you can disable these CREF warnings via `#pragma warning disable CS0419 // Ambiguous reference in cref attribute` while reviewing https://github.com/akkadotnet/akka.net/pull/4629 - that might be helpful in some instances where the cref is actually useful but can't be resolved.

**Comment by Aaronontheweb** [comment]
@eaba that comment is interesting - maybe we need to modify the DocFx build step in FAKE to do `buildRelease` first THEN compile the documentation? Give that a try?

**Comment by eaba** [comment]
> @eaba that comment is interesting - maybe we need to modify the DocFx build step in FAKE to do `buildRelease` first THEN compile the documentation? Give that a try?

I think @Arkatufus should look into how https://github.com/akkadotnet/akka.net/pull/5542 uses docfx. Because on my machine, the warning are not there! 

Right now the only way forward is to work with the warnings I got from running `build.cmd docfx` locally.

**Comment by Aaronontheweb** [comment]
> > @eaba that comment is interesting - maybe we need to modify the DocFx build step in FAKE to do `buildRelease` first THEN compile the documentation? Give that a try?
> 
> I think @Arkatufus should look into how #5542 uses docfx. Because on my machine, the warning are not there!
> 
> Right now the only way forward is to work with the warnings I got from running `build.cmd docfx` locally.

You can look at the PR and see what he's doing there

**Comment by Aaronontheweb** [comment]
No longer needed as a result of DocFx build changes per @eaba

**Pull request closed without merging**